src/main/java/persistence/entity/EntityManagerFactory.java

```diff
@@ -1,28 +1,24 @@
 package persistence.entity;
 
 
-import java.util.Set;
+import java.sql.Connection;
 import jdbc.JdbcTemplate;
-import persistence.dialect.Dialect;
+import persistence.meta.MetaModel;
 
 
 public class EntityManagerFactory {
-    private final EntityPersisteContext entityPersisteContext;
-
-    private EntityManagerFactory(Set<Class<?>> entityClasses, JdbcTemplate jdbcTemplate, Dialect dialect) {
-        this.entityPersisteContext = EntityPersisteContext.create(entityClasses, jdbcTemplate, dialect);
-    }
-
-    public static EntityManagerFactory of(String packageName, JdbcTemplate jdbcTemplate, Dialect dialect) {
-        return new EntityManagerFactory(getEntityClassesFromPackage(packageName), jdbcTemplate, dialect);
+    private MetaModel metaModel;
```

Comment:
사소하지만 변경되지 않는 변수는 final 을 붙여주시면 좋을 것 같아요 😄 

```suggestion
    private final MetaModel metaModel;
```

Reply:
앗 놓쳤습니다. 변경해야겠어요